src/app/components/mno-select-app-wizard/mno-select-app-wizard.html

```diff
@@ -4,7 +4,7 @@
       <div class="recommendation-sentence" ng-show="isRecommendationShown">{{ 'mno_enterprise.templates.onboarding.select_your_app.we_recommend' | translate }}</div>
       <mno-widget>
         <mno-widget-header>
-          <input type="text" ng-model="searchTerm.name" placeholder="{{ 'mno_enterprise.templates.onboarding.select_your_app.search' | translate }}" class="form-control input-sm search-bar" ng-change="selectedCategory.category = ''">
+          <input type="text" ng-model="searchTerm.name" placeholder="&#xF002; {{ 'mno_enterprise.templates.onboarding.select_your_app.search' | translate }}" class="form-control input-sm search-bar" ng-change="selectedCategory.category = ''">
```

Comment:
Using a `input-group-addon` would probably be nicer than using this unicode character:
http://getbootstrap.com/css/#forms